css-editor/src/Background.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+
+export default class Background extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    }
```

Comment:
If states are empty you can leave it out since React would still create states for you when you use setState()